backend/src/src-cvesearch/src/test/java/org/eclipse/sw360/cvesearch/TestWithCveSearchConnection.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ * With modifications by Siemens AG, 2016.
+ * Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.cvesearch;
+
+import org.eclipse.sw360.cvesearch.datasource.CveSearchApi;
+import org.eclipse.sw360.cvesearch.datasource.CveSearchApiImpl;
+import org.eclipse.sw360.datahandler.common.CommonUtils;
+import org.eclipse.sw360.cvesearch.service.CveSearchHandler;
+import org.junit.Assume;
+import org.junit.Before;
+
+import java.util.Properties;
+
+import static org.eclipse.sw360.cvesearch.datasource.CveSearchDataTestHelper.isUrlReachable;
+
+public abstract class TestWithCveSearchConnection {
+    private String PUBLIC_CVE_SEARCH_SERVER = "https://cve.circl.lu";
+
+    protected CveSearchApi cveSearchApi;
+
+    @Before
+    public void setUpApi() {
+        Properties props = CommonUtils.loadProperties(TestWithCveSearchConnection.class, "/cvesearch.properties");
+        String host = props.getProperty(CveSearchHandler.CVESEARCH_HOST_PROPERTY, "https://cve.circl.lu");
+
+        Assume.assumeTrue("The public CVE Search server is currently unreliable and tests are only executed agains other instances.", ! PUBLIC_CVE_SEARCH_SERVER.equals(host));
```

Comment:
Isn't it testing that you have configured another CVE server than the public one? Therefore the message sould be someting like "Another then the public server has been configured. ...."

Reply:
if the default public server is used, `! PUBLIC_CVE_SEARCH_SERVER.equals(host)` is false and thus triggers the assume statement. This then says that the tests are skipped since the default public server is not reliable.

Reply:
Thanks for clarification! So the statement is: we never test against the public CVE server because it is too unstable.

I think the currently has mislead me because it sounds like: we've checked the public server right now and unfortunately it is **currently** unreliable. So for **this** time we skip the tests.

Maybe remove it. Then it is clear that we think that the public server is unreliable in general.